### PR TITLE
Lab 01 first draft

### PR DESCRIPTION
**Pull request by trstephen**
- [x] TODO: Some words about how the results relate to the prelab calcs

### CODE REVIEW

File: lab01/sections/results.tex [hunk 1]
```diff
@@ -1,3 +1,213 @@
...
+Section \ref{sec:bump} uses measured system input and output under dynamic conditions to determine the transfer function parameters.
+The parameters derived from the prelab, static conditions and dynamic conditions are compared in Section \ref{sec:validation}.
+
+\subsection{Initial experimental tests}\label{sec:initial}
+The motor described in the lab manual \cite{lab-manual} is meant to be roughly equivalent to the motor used in the experiment.
+One notable difference is that the lab manual motor has a maximum $u_m$ of \SI{15}{\volt} DC.
+The QICii software used to control the experiment motor only allows a maximum $u_m$ of \SI{5}{\volt} DC.
+To establish the validity of the lab manual motor parameters as approximations for the experiment motor we will compare the $\omega_{m\>max}$ from each source.
+
+Using KVL in Figure~\ref{fig:motorschematic}, we can derive the relationship between the input voltage and the motor angular velocity as
```
Comment (trstephen) [inline lab01/sections/results.tex]: here is a useful comment

**Comment by trstephen** [comment]
Printing this off and submitting around 9:30.

File: lab01/sections/intro.tex [hunk 2]
```diff
@@ -1,3 +1,52 @@
...
-References are appreciated \cite{lab-manual}.
+This experiment will develop a model for a DC motor controller.
+The angular velocity of a DC motor in an armature configuration, shown in Figure~\ref{fig:motorschematic}, can be approximated by a first order system.
+We can vary the applied voltage $u_m$ in a static configuration to determine the internal resistance of the particular motor $R_m$ and its torque constant $k_m$.
+These two values let us determine the the gain $K$ and time constant $\tau$ of the governing first order equation
+\begin{equation}\label{eq:tf}
+  G(s) = {K \over \tau s + 1}.
+\end{equation}
+This model will be improved by analyzing the system input and output under dynamic conditions.
+Measurements in dynamic conditions make fewer assumptions about the system and give a better approximation for $K$ and $\tau$.
```
Comment (SilvanoT) [inline lab01/sections/intro.tex:10]: In the draft the tau is written as "tau", and in the previous paragraph it is the symbol for tau. These should be the same, as the symbol, for consistency in my opinion.

Comment (trstephen) [reply lab01/sections/intro.tex:10]: Can you link the line you're referring to? Searching the doc shows that "tau" only appears as `$\tau$` in text. `\tau` can be used in math mode (equation and align environments) without the `$` delimeters.

Comment (trstephen) [reply lab01/sections/intro.tex:10]: Well... this is handed in so I might as well merge 👀

**Pull request merged**